components/project-loader/project-loader.jsx

```diff
@@ -130,12 +130,23 @@ export default React.createClass({
 
     return <p>{searchResultNotice}</p>;
   },
+  renderIssueCountResult() {
+    if(this.props.latest) return null; 
+    if(this.props.featured || this.props.search || this.state.loadingData) return null;
+    
+    let total = this.state.totalMatched,
+      plural = (total === 0 || total > 1), // because "0 results"
+      issueCountResult = `${total} result${plural ? `s` : ``} found`;
+      
+    return <p>{issueCountResult}</p>;
+  },
```

Comment:
from https://github.com/mozilla/network-pulse/issues/517#issuecomment-302813765 we are asked to display The counter on individual issue pages and tag pages **only**.

I would change the above code to something like

```jsx
renderEntryCounter() {
  if (this.state.loadingData) return null;
  if (!this.props.issue && !this.props.tag) return null;

  return <p>{this.state.totalMatched} result{this.state.totalMatched > 0 ? `s` : ``}</p>;
}
```